packages/components/src/internal/components/assay/AssayDesignUploadPanel.tsx

```diff
@@ -44,6 +44,7 @@ export const AssayDesignUploadPanel: FC<AssayDesignUploadPanelProps> = memo(prop
                     />
                 </Col>
             </Row>
+            {children}
```

Comment:
Yeah I was thinking the same just was trying to avoid a bigger refactor. I'll check out moving the container select to the tab components

Reply:
I didn't factor it into the tab components just because it would add the five or so same props to each, instead left it as child but memoized it to a single instance of the container select

Reply:
Sounds good. Thanks.